### PR TITLE
feat: implement code for Array Operations

### PR DESCRIPTION
**Pull request by ArielRojas**
image

### CODE REVIEW

File: Generics/ArrayOperations.cs [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    /// Finds the maximum value.
+    /// </summary>
+    /// <param name="array">Array with the values.</param>
+    /// <returns>The maximum value.</returns>
+    public static T FindMaximum(T[] array)
+    {
+        if (array == null || array.Length == 0)
+        {
+            return default;
+        }
```
Comment (bernarth) [inline Generics/ArrayOperations.cs:19]: Don't forget to leave a space fot the `return`

File: Generics/ArrayOperations.cs [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    /// </summary>
+    /// <param name="array">Array with the values.</param>
+    /// <returns>The minimum value.</returns>
+    public static T FindMinimum(T[] array)
+    {
+        if (array == null || array.Length == 0)
+        {
+            return default;
+        }
+        return array.Min();
```
Comment (bernarth) [inline Generics/ArrayOperations.cs:37]: here too

**Review by bernarth — APPROVED** [review]
Good job

File: Generics/ArrayOperations.cs (hunk 2 above)
Comment (ArielRojas) [reply Generics/ArrayOperations.cs:37]: Ok, I added it.

File: Generics/ArrayOperations.cs (hunk 1 above)
Comment (ArielRojas) [reply Generics/ArrayOperations.cs:19]: Ok, I added it.

**Pull request merged**